ddtrace/profiling/collector/_task.pyx

```diff
@@ -49,3 +64,25 @@ cpdef get_task(thread_id):
         frame = None
 
     return task_id, task_name, frame
+
+
+cpdef list_tasks():
+    # type: (...) -> typing.List[typing.Tuple[int, str, types.FrameType]]
```

Comment:
do you have a `.pyi` file for this that should get updated as well?

these type comments don't get picked up by mypy right?

Reply:
Added them manually. Indeed, `pyx` files with `cpdef` are not really parsable by mypy 😢